src/platforms/linux/xdgportal.h

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef XDGPORTAL_H
+#define XDGPORTAL_H
+
+#include <QObject>
+
+constexpr const char* XDG_PORTAL_SERVICE = "org.freedesktop.portal.Desktop";
+constexpr const char* XDG_PORTAL_PATH = "/org/freedesktop/portal/desktop";
+
+constexpr const char* XDG_PORTAL_BACKGROUND =
+    "org.freedesktop.portal.Background";
+constexpr const char* XDG_PORTAL_REQUEST = "org.freedesktop.portal.Request";
+constexpr const char* XDG_PORTAL_SECRET = "org.freedesktop.portal.Secret";
+
+class XdgPortal : public QObject {
```

Comment:
Nit: Could you add a few lines of Doc text about this class? 
as i'm not familiar with portals that would be nice 😅

Reply:
Will do